razorpay-payments.php

```diff
@@ -527,14 +572,19 @@ function check_razorpay_response()
             {
                 $success = false;
                 $error = 'Customer cancelled the payment';
-
                 $this->handleErrorCase($order);
             }
 
             $this->updateOrder($order, $success, $error, $razorpayPaymentId);
 
+            $this->redirectUser($order);
+        }
+
+        protected function redirectUser($order)
+        {
```

Comment:
Won't the order be updated twice in that case? The payment note gets added twice, right?